EmployeeDirectory.DAL/DataOperations/EmployeeOperations.cs

```diff
@@ -0,0 +1,174 @@
+﻿using EmployeeDirectory.DAL.Contracts.Providers;
+using EmployeeDirectory.DAL.Exceptions;
+using EmployeeDirectory.DAL.Models;
+using Microsoft.Data.SqlClient;
+using Microsoft.IdentityModel.Tokens;
+
+namespace EmployeeDirectory.DAL.DataOperations
+{
+    public class EmployeeOperations(IDBConnection dBConnection):IEmployeeOperations
```

Comment:
Rename all DataOperation classes to its specific repository and also change the name of folder.